packages/components/shared/src/controllers/anchor.ts

```diff
@@ -34,6 +35,10 @@ export class AnchorController implements ReactiveController {
   #onToggle = (event: Event): void => {
     const { newState, oldState, target } = event as ToggleEvent & { target: HTMLElement };
 
+    if (this.#host instanceof Tooltip) {
```

Comment:
Hmm, doesn't this create a circular dependency? Perhaps better to do `if (this.#host.tagName === 'SL-TOOLTIP') {` ?

Reply:
Right